### PR TITLE
Napiprojekt provider new options:

### PR DESCRIPTION
**Pull request by destpstrzy**
Napiprojekt is big polish subtitle base but there are a lot of AI generated subtitles uploaded by random people. I have done some improvements that should eliminate limit incorrect downloads:

Napiprojekt provider new options:
- Info about releases
- Skip subtitles without authors or possibly AI generated
- Download subtitles with real name authors only

### CODE REVIEW

**Comment by morpheus65535** [comment]
Could you rebase this PR on development branch please?

https://github.com/morpheus65535/bazarr/blob/master/CONTRIBUTING.md#community-devs

**Comment by destpstrzy** [comment]
> Could you rebase this PR on development branch please?
> 
> https://github.com/morpheus65535/bazarr/blob/master/CONTRIBUTING.md#community-devs

Done. Sorry for the mistake.

File: frontend/package-lock.json
Comment (morpheus65535) [inline frontend/package-lock.json:1]: Don't bring this with you please.

File: frontend/package.json
Comment (morpheus65535) [inline frontend/package.json:1]: Don't bring this with you please.

File: changelog-master.hbs
Comment (morpheus65535) [inline changelog-master.hbs:1]: Don't bring this with you please.

File: changelog-dev.hbs
Comment (morpheus65535) [inline changelog-dev.hbs:1]: Don't bring this with you please.

**Review by morpheus65535 — CHANGES_REQUESTED** [review]

**Review by morpheus65535 — APPROVED** [review]

**Pull request merged**